runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/windmill/work/refresh/Heartbeats.java

```diff
@@ -45,21 +46,32 @@ public abstract static class Builder {
 
     abstract ImmutableList.Builder<RefreshableWork> workBuilder();
```

Comment:
can this be private?

Reply:
it cannot as we rely on AutoValue.Builder to generate the code